Squablr/Views/CardView.m

```diff
@@ -25,6 +25,89 @@ - (instancetype)initWithFrame:(CGRect)frame {
     return self;
 }
 
+- (instancetype)initWithProfile:(CGRect)frame profile:(Profile *)profile {
```

Comment:
Need to change the name to `initWithFrame:(CGRect)frame profile:(Profile *)profile`